test/EFCore.PG.FunctionalTests/NpgsqlMigrationSqlGeneratorTest.cs

```diff
@@ -1016,6 +1016,41 @@ public void CreateIndexOperation_nulls_first()
 ");
         }
 
+        [Fact]
+        public void CreateIndexOperation_tsvector()
```

Comment:
Since you did this PR, I've merged support for new, functional migration tests which actually run against the database (as opposed to just asserting on SQLs), see https://github.com/dotnet/efcore/issues/19039 for more details.

Can you please redo these tests in MigrationsNpgsqlTest? It should be pretty easy, just follow one of the other Create_index tests.

Reply:
Done.